frontend/src/console/components/ThemeNavigation/ThemeNavigation.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import './styles.scss';
+import {
+  ConsolePage,
+  ConsolePageCustomizationContainer,
+  NavigationMenu
+} from 'console/components';
+import { InstancesModel } from 'console/models';
+import { connect } from 'react-redux';
+import { RootState } from 'global/state';
+import { updateThemeFieldValue } from 'console/actions';
+import { Col, Row } from 'react-bootstrap';
+import { ColorInputField } from '../../../ui/components/ColorInputField';
+import messages from './displayMessages';
+import { WrappedMessage } from '../../../utils/intl';
+
+interface State {}
+
+interface ActionProps {
+  updateThemeFieldValue: Function;
+}
+
+interface StateProps extends InstancesModel {}
+
+interface Props extends StateProps, ActionProps {}
+
+export class ThemeNavigationComponent extends React.PureComponent<
+  Props,
+  State
+> {
+  private onChangeColor = (fieldName: string, newColor: string) => {
+    const instance = this.props.activeInstance;
+
+    if (instance.data) {
+      this.props.updateThemeFieldValue(instance.data.id, fieldName, newColor);
+    }
+  };
+
+  public render() {
+    const instance = this.props.activeInstance;
+    let themeData;
+
+    if (instance.data && instance.data.draftThemeConfig) {
+      themeData = instance.data.draftThemeConfig;
+    }
+
+    return (
+      <ConsolePage contentLoading={this.props.loading}>
+        <ConsolePageCustomizationContainer>
+          <h2>
+            <WrappedMessage messages={messages} id="themeNavigation" />
+          </h2>
+
+          {themeData && themeData.version === 1 && (
+            <div className="theme-navigation-container">
+              <NavigationMenu themeData={themeData} />
+              <NavigationMenu themeData={themeData} loggedIn />
+              <Row>
+                <Col>
```

Comment:
@Agrendalath Can you try using `<Col md={3}>` for each field?

Reply:
@giovannicimolin, I already tried this, but the vertical alignment of the input fields looked odd to me due to text wrapping. I just fixed it with `align-items: end`.

This is how it looks with `md={4}`:
![md4](https://user-images.githubusercontent.com/3189670/77448648-56430480-6df1-11ea-94d2-b08c9838dcb4.png)

This is how it looks with `md={3}`:
![md3](https://user-images.githubusercontent.com/3189670/77448671-5c38e580-6df1-11ea-83be-ae2cbc26d2e9.png)

For now I left `md={4}`, as it looked more readable to me. What do you think?

Reply:
Alignment looks great now!